### PR TITLE
app-emacs/{markdown-mode,scala-mode,distel,moccur-edit}: use HTTPS

### PR DESCRIPTION
**Pull request by mm1ke**
Hi,

This PR fixes a few app-emacs packages to use https instead of http.

Please review.

### CODE REVIEW

File: app-emacs/scala-mode/scala-mode-2.10.3.ebuild [hunk 1]
```diff
@@ -7,8 +7,8 @@ inherit elisp
 
 MY_P="scala-tool-support-${PV}"
 DESCRIPTION="Scala mode for Emacs"
-HOMEPAGE="http://www.scala-lang.org/"
-SRC_URI="http://www.scala-lang.org/files/archive/${MY_P}.tgz"
+HOMEPAGE="https://www.scala-lang.org"
```
Comment (ulm) [inline app-emacs/scala-mode/scala-mode-2.10.3.ebuild]: Please don't remove the trailing slash from URLs.

**Comment by ulm** [comment]
So why was this merged, in spite of changes being requested?
Reverted now.

**Comment by flappyports** [comment]
@ulm Meh, all that energy to revert a commit.  You could have just added your preferred trailing slash back and "fixed" two things at once.

**Comment by mm1ke** [comment]
@ulm Updated this PR, should be fine now.

**Comment by ulm** [comment]
Merged.

**Pull request closed without merging**